cardano-api/src/Cardano/Api/Shelley.hs

```diff
@@ -204,8 +204,12 @@ module Cardano.Api.Shelley
     ProtocolState(..),
     decodeProtocolState,
     SerialisedDebugLedgerState(..),
+    CurrentEpochState(..),
```

Comment:
There's tension between keeping things in alphabetical order and keeping related things together and also how ordering should occur between lower and upper case.

It's unlikely to me how to make this tradeoff.  It might be worth agreeing on some rules on how to make this tradeoff.

Reply:
Sure, let's revisit this in the future.